### PR TITLE
feat(command-bar): add commands from existing palette

### PR DESCRIPTION
**Pull request by thmsobrmlr**
## Problem

Follow up to https://github.com/PostHog/posthog/pull/17864.

## Changes

This PR adds a CMD+SHIFT+K menu displaying the commands from the existing CMD+K palette.

Follow up work: https://github.com/PostHog/posthog/issues/18552

## How did you test this code?

Tested locally

### CODE REVIEW

File: frontend/src/lib/components/CommandBar/ActionResults.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    return (
+        <div className="grow overscroll-none overflow-y-auto">
+            {commandSearchResultsGrouped.map(([scope, results]) => (
+                <ResultsGroup key={scope} scope={scope} results={results} activeResultIndex={activeResultIndex} />
+            ))}
+        </div>
+    )
+}
+
+export default ActionResults
```
Comment (benjackwhite) [inline frontend/src/lib/components/CommandBar/ActionResults.tsx:42]: Personally I would like to not have any `export default` as it makes the auto-importing harder

File: frontend/__snapshots__/scenes-app-insights--retention--webkit.png
Comment (daibhin) [inline frontend/__snapshots__/scenes-app-insights--retention--webkit.png:1]: The placeholder didn't render when doing CMD+K (it did with CMD+SHIFT+K)
placeholder

Comment (daibhin) [inline frontend/__snapshots__/scenes-app-insights--retention--webkit.png:1]: Highlighting all the text and replacing it with `>` does not recognise that it should enter the actions mode
trigger

Comment (daibhin) [inline frontend/__snapshots__/scenes-app-insights--retention--webkit.png:1]: It isn't clear what this action does (I assume it opens the toolbar). Do we need some kind of description?
Screenshot 2023-11-13 at 09 57 04

Comment (daibhin) [inline frontend/__snapshots__/scenes-app-insights--retention--webkit.png:1]: The blank space below the options looked weird to me. Maybe it should be the same color as the options Screenshot 2023-11-13 at 09 56 23

**Review by daibhin — COMMENTED** [review]
I just left some UX comments on the first file randomly so that you can reply individually if needed. I know you wanted to get this out and follow up with UX fixes afterwards so nothing blocking from me

**Review by benjackwhite — COMMENTED** [review]
Quick skim and all looked fine. Other than a comment I will try and check it out and try it out myseld

**Pull request merged**